splinterd/tests/admin/circuit_lifecycle.rs

```diff
@@ -38,7 +37,7 @@ use splinterd::node::RestApiVariant;
 fn make_create_circuit_payload(
     circuit_id: &str,
     requester: &str,
-    node_info: HashMap<String, String>,
+    node_info: &[(String, String)],
```

Comment:
If you changed this to a `&mut dyn Iterator<Item = (...)>`, you would have had to change far less code.